plugins/nodes/src/js/components/NodesGridView.js

```diff
@@ -86,8 +86,7 @@ var NodesGridView = React.createClass({
 
     // Add 'Others' node to the list
     if (activeServiceIds.length > MAX_SERVICES_TO_SHOW) {
-      var classNameOther =
-        `dot service-color-${OTHER_SERVICES_COLOR}`;
+      var classNameOther = `dot service-color-${OTHER_SERVICES_COLOR}`;
```

Comment:
prettier? :D

Reply:
I can't really see a difference in the UI. 🤔 https://cl.ly/3A1A2Y0m093u